SIR-X.ipynb

```diff
@@ -541,8 +578,20 @@
    },
    "outputs": [],
    "source": [
-    "# Get the confidence interval through bootstrap\n",
-    "par_ci, par_list = ci_bootstrap(SIR_UK, t_d, I_UK, P_UK, n_iter=1000)"
+    "# Get the confidence interval through random bootstrap\n",
```

Comment:
Why are there so many changes in non-tests files?

Reply:
Ah, I think it has to do with this: https://github.com/open-sir/open-sir/pull/23#issuecomment-609497860